### PR TITLE
fix: update prefect version in worker package workflow

### PR DESCRIPTION
**Pull request by jamiezieziula**

### CODE REVIEW

**Review by jimid27 — DISMISSED** [review]

File: .github/workflows/helm-release.yaml [hunk 1]
```diff
@@ -78,6 +78,8 @@ jobs:
         run: |
           mkdir -p /tmp/chart
           cd charts
+          # Update the prefect version tag in values.yaml
+          sed -i "s/prefectTag:.*$/prefectTag: $PREFECT_VERSION-python3.11/g" prefect-worker/values.yaml
```
Comment (jawnsy) [inline .github/workflows/helm-release.yaml:82]: Does the version include the kubernetes tag? I'm assuming yes

**Review by jawnsy — DISMISSED** [review]

File: .github/workflows/helm-release.yaml (hunk 1 above)
Comment (jamiezieziula) [reply .github/workflows/helm-release.yaml:82]: oh - it doesnt!

Comment (jamiezieziula) [reply .github/workflows/helm-release.yaml:82]: well actually im not sure let me check

**Review by jimid27 — APPROVED** [review]
lgtm

**Pull request merged**